### ISSUE TITLE
Add parent and ancestor arguments to pages query

### ISSUE THREAD

**Issue by dopry** [torchbox/wagtail-grapple#239]
It would be nice to subselection based on where pages are in the site tree, In my specific use case I'm building an SPA where we have a 'stories' landing page, and need to show a paginated list of stories that are children of the 'stories' page.

alternatively, they could probably be named parent and ancestor...

**Comment by dopry** [comment]
I didn't realize there were children and descendants fields. If you're looking to do something similar you should query the children or descendants of your root page, rather than trying to treat pages as a flat list using a child_of or descendant_of filter.

```graphql
query {
  pages (id: 3) {
    id
    title
    parent {
      id
    }
    children {
      id
    }
    descendants {
      id
    }
  }
}
```

**Comment by zerolab** [comment]
This is in the docs - https://wagtail-grapple.readthedocs.io/en/latest/general-usage/graphql-types.html?highlight=descendants#pageinterface

There's scope to improve the documentation

**Comment by dopry** [comment]
This was implemented in #240
